honeybee_grasshopper_energy/src/HB Blind Material.py

```diff
@@ -0,0 +1,92 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Create a material for a blind layer in a window construction.
+This material can be plugged into the "HB Window Construction" component.
+_
+Window blind properties consist of flat, equally-spaced slats.
+-
+
+    Args:
+        _name: Text string for material name.
+        _vertical_: Set to "True" to have the blinds be vertically-oriented and
```

Comment:
I think it is cleaner to have it as orientation and then 0 and 1 can be mapped to vertical versus horizontal.

Reply:
`orientation` is what it is called when its a text input like `"Horizontal"` or `"Vertical"`.  When it's `True` or `False`, it has always seemed clearer to me to label the input with what the `True` value corresponds to.  I have tried to follow this convention throughout the Python API for the few cases where I accept Boolean inputs instead of Text or I am mapping boolean inputs to IDF Text.  If you think we should just throw out this convention and not distinguish between cases of Text input vs Boolean input, let me know and I'll change it all to be what we follow for the text input.

Reply:
It is a design decision and I agree that both can work. I personally like not using the options as the input name but that's a personal choice. Using it as is will work just as fine.